internal/gen/gen.go

```diff
@@ -30,6 +30,10 @@ func GoFmt(i Interface) Interface {
 		if err != nil {
 			return nil, err
 		}
-		return format.Source(b)
+		formatted, err := format.Source(b)
+		if err != nil {
+			formatted = b
```

Comment:
nit: assigning `b` to a variable called `formatted` bothers me, since `b` is not formatted. Just return directly?

```suggestion
			return b, err
```